luigi/contrib/hadoop.py

```diff
@@ -478,6 +479,12 @@ def run_job(self, job, tracking_url_callback=None):
         if libjars:
             arglist += ['-libjars', ','.join(libjars)]
 
+        # 'archives' is also a generic option
+        archives = [archive for archive in self.archives]
```

Comment:
I was wondering the same to be honest. I mimicked the `libjars` variable behavior.